pip install preshed fails for official python:3.7 docker container

Hi, I'm trying to install spacy in the official [python:3.7](https://store.docker.com/images/python) docker container but I'm getting an error: `Failed building wheel for cymem`.

Here's the command and the [full log](https://pastebin.com/D8xk6y5y):

```
docker run -it --rm python:3.7 bash -c 'pip install preshed'
```

Note that running this command using the `python:3.6` container works fine.

Let me know if any additional information would be useful!

preshed fails at
```
    preshed/maps.cpp:6647:13: error: ‘PyThreadState {aka struct _ts}’ has no member named ‘exc_traceback’; did you mean ‘curexc_traceback’?
         tstate->exc_traceback = *tb;
                 ^~~~~~~~~~~~~
                 curexc_traceback
    error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
```
which (googleficially) looks like https://github.com/pandas-dev/pandas/issues/21785

--- a/.travis.yml
+++ b/.travis.yml
@@ -9,6 +9,7 @@ python:
   - "3.4"
   - "3.5"
   - "3.6"
+  - "3.7-dev"
 
 install:
 - pip install cython

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,3 +1,3 @@
 cymem>=1.30,<1.31
-cython >= 0.23
+cython >= 0.28
 pytest

